woocommerce/woocommerce-android
# Add analytics to track device type on some screens

Closes: #10806 


### Description

This PR adds analytics to track the device type to analyse the impact of project Lily-Pad M2 changes, it would be useful to know when the two pane layout is in use.

We can do that by tracking the horizontal size class – this will be either compact (no split) or regular (split view shown)

### Testing instructions


1. Launch the app
2. Tap the Orders tab
3. Observe that the main_tab_orders_selected event is tracked with a horizontal_size_class property, matching your current size class.
4. Repeat for the main_tab_orders_reselected event (tap the Orders tab again)
5. Also check order_open (select an order), order_create_button_tapped (create an order), and payments_flow_order_collect_payment_tapped (create an order by tapping Collect Payment).
